.kokoro/android/common.cfg

```diff
@@ -0,0 +1,18 @@
+# Format: //devtools/kokoro/config/proto/build.proto
+
+# The github token is stored here.
+gfile_resources: "/bigstore/cloud-devrel-kokoro-resources/dpebot"
+
+# Common env vars for all repositories and builds.
+env_vars: {
+    key: "DPEBOT_GITHUB_TOKEN_FILE"
+    value: "github-token.txt"
+}
+env_vars: {
+    key: "DPEBOT_GIT_USER_NAME"
+    value: "DPE bot"
```

Comment:
Is this correct?

Reply:
Oh it works for Firebase so ... guess it doesn't matter.

Reply:
I copied the [firebase common](https://github.com/GoogleCloudPlatform/repository-gardener/blob/master/.kokoro/firebase/common.cfg)... maybe it isn't used?